packages/system/immucore/collection.yaml

```diff
@@ -21,3 +21,15 @@ packages:
       - https://github.com/kairos-io/immucore
     license: "Apache License v2"
     description: "The Kairos immutability management interface"
+  - name: "immucore"
+    category: "fips-static"
```

Comment:
should fips static just be fips? I mean, does it help to have 2 different fips binaries when we could have one that should work everywhere?

Reply:
I don't know if it has implication in runtime yet, hence why a new category